### ISSUE TITLE
change debian file system

### ISSUE THREAD

**Issue by omymy1** [GoogleContainerTools/distroless#793]
what file system debian image use?
how to change debian filesystem in using dockerfile?
I want to use ReiserFS file system

**Comment by polarathene** [comment]
> what file system debian image use?

Docker images and container runtime storage is written to the host filesystem with the container writes handled by the [Docker daemon storage driver you have configured](https://docs.docker.com/storage/storagedriver/#container-and-layers) (_default likely `overlay2`_).

> how to change debian filesystem in using dockerfile?

You don't.

You can change you find another supported storage driver is more suitable than the default one Docker is using on your system.

You also have the option of anonymous, named and bind mount volumes that you can use to connect storage to your running container. The bind mount volume will mount a file or directory from the host filesystem and read/write to that.

> I want to use ReiserFS file system

There is no storage or volume driver for that. You should have this filesystem setup on your host system and use a bind mount volume to it from your container (_eg: via the CLI with `docker run` or `docker-compose.yml` config, see official docs for either of those_).

---

## TL;DR

This project and Dockerfile have nothing to do with what filesystem is used by your container or the image layer storage.
